### PR TITLE
Fixed issue #1, Inconsistent Sidebar Styles

### PR DESCRIPTION
**Pull request by im-mattbrown**
This commit fixes Issue #1  the inconsistent sidebar styling where the class name for div and h3 in lib/archives_sidebar/app/views/archives_sidebar/_content.html.erb. 

There is no css class for "sidebar_title" or "sidebar_body" and they should have been class "sidebar-body" instead.

### CODE REVIEW

File: lib/archives_sidebar/lib/archives_sidebar.rb [hunk 1]
```diff
@@ -30,7 +30,7 @@ def parse_request(_contents, _params)
     article_counts = Content.find_by_sql(["select count(*) as count, #{date_func} from contents where type='Article' and published = ? and published_at < ? group by year,month order by year desc,month desc limit ? ", true, Time.now, count.to_i])
 
     @archives = article_counts.map do |entry|
-      month = (entry.month.to_i%12)+1
+      month = (entry.month.to_i % 12) + 1
```
Comment (nathanallen) [inline lib/archives_sidebar/lib/archives_sidebar.rb:33]: Please explain this change, and how it relates to the sidebar styles.

Comment (im-mattbrown) [reply lib/archives_sidebar/lib/archives_sidebar.rb:33]: This change has no relation to the sidebar styles, however, the directions ask that we run Rubocop and this was flagged as an error for missing space between operators.

Comment (nathanallen) [reply lib/archives_sidebar/lib/archives_sidebar.rb:33]: Good catch! Given that the rubo-cop errors are in a different area of code, and not related to your changes, please pull them out. If you'd like to create a new issue/pr for the linting issue, we would welcome it!

**Comment by nathanallen** [comment]
We would prefer you not close PRs so that we do not lose discussion history. At a minimum please reference this PR in your new one.

**Pull request closed without merging**